tox.ini

```diff
@@ -23,7 +23,7 @@ ignore=
 usedevelop=True
 commands=
     core: pytest {posargs:tests/core}
-    docs: make build-docs
+    docs: make check-docs
```

Comment:
Nice change to get the newsfragment validation into CI 👍🏻 -- I'm a little surprised, because I thought for sure that I've seen newsfragments failures cause CI to turn red before, but not sure how...

Ah, it's because py-evm was explicitly calling `validate-docs`:
https://github.com/ethereum/py-evm/blob/7674e2e0caefa8b1208f650bb71a07c65ef2f305/tox.ini#L65-L73